schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_entry.json

```diff
@@ -10289,6 +10289,894 @@
          "minItems": 1,
          "uniqueItems": true
       },
+      "pdbx_vrpt_summary": {
+         "type": "object",
+         "properties": {
+            "EMDB_deposition_date": {
```

Comment:
How about suppressing it? In light of upcoming incorporation of EMDB data

Reply:
@valasatava Should we keep this until EMDB data is incorporated?